recipe/meta.yaml

```diff
@@ -9,8 +9,14 @@ source:
   sha256: 13f93ce9bea8016c253b34afc6bd6a75993e5c40672ed5405a9c832f0d4a00bc
 
 build:
-  number: 0
+  number: 1
   skip: true  # [py<37 or win32]
+  # linux-ppc64le failing test phase on all 1.1.1 builds skipping until 1.1.1 is dropped completely
+  # for more info on error: https://anaconda.atlassian.net/browse/PKG-2252
+  # This can be dropped after OpenSSL 1.1.1 support is dropped in Sept 2023
+  {% if ARCH == "ppc64le" and (openssl | string).startswith('1.1.1') %}
```

Comment:
I'm struggling to understand why this is needed. We don't build for OpenSSL 1 anymore.

Reply:
I tried building without and it failed. https://cloud.prefect.io/anaconda/flow-run/24d55d8f-cbee-437b-b007-b126c499ac7b?logs